.vscode/settings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I think this should be in .gitignore

Reply:
This is interesting as the .gitignore considers the `settings.json` to not be ignored. It seems that intention of the person who started this project was to consider settings.json as part of the git repo.
Anyway, removing my changes.